packages/@aws-cdk/cdk/lib/stack.ts

```diff
@@ -116,15 +119,16 @@ export class Stack extends Construct implements ITaggable {
   public readonly name: string;
 
   /**
-   * Should the Stack be deployed when running `cdk deploy` without arguments
-   * (and listed when running `cdk synth` without arguments).
-   * Setting this to `false` is useful when you have a Stack in your CDK app
-   * that you don't want to deploy using the CDK toolkit -
-   * for example, because you're planning on deploying it through CodePipeline.
+   * Indicates if this stack is hidden, which means that it won't be
+   * automatically deployed when running `cdk deploy` without arguments.
+   *
+   * Setting this to `true` is useful when you have a stack in your CDK app
+   * which you don't want to deploy using the CDK toolkit. For example, because
+   * you're planning on deploying it through a deployment pipeline.
    *
-   * By default, this is `true`.
+   * By default, all stacks are visible (this will be false).
```

Comment:
I wouldn't even put this line. The `StackProps` comment is enough. Also, my naming dislike applies here, too, obviously.

Reply:
Not sure why we're mirroring input properties to output properties at all. Shouldn't this be removed?

Reply:
made private